tests/app/notifications/test_receive_notification.py

```diff
@@ -37,14 +37,21 @@ def firetext_post(client, data, auth=True, password='testkey'):
     )
 
 
-def mmg_post(client, data):
+def mmg_post(client, data, auth=True, password='testkey'):
+    headers = [
+        ('Content-Type', 'application/json'),
+        ('X-Forwarded-For', '203.0.113.195, 70.41.3.18, 150.172.238.178')
+    ]
+
+    if auth:
+        auth_value = base64.b64encode("notify:{}".format(password).encode('utf-8')).decode('utf-8')
+        headers.append(('Authorization', 'Basic ' + auth_value))
+
     return client.post(
         path='/notifications/sms/receive/mmg',
         data=json.dumps(data),
-        headers=[
-            ('Content-Type', 'application/json'),
-            ('X-Forwarded-For', '203.0.113.195, 70.41.3.18, 150.172.238.178')
-        ])
+        headers=headers
+    )
 
 
 def test_receive_notification_returns_received_to_mmg(client, mocker, sample_service_full_permissions):
```

Comment:
The if and else blocks are not really being tested can you mock the logger and check they are being called?

Reply:
Only wants to observe the blocking at the moment. If we take away the commented blocking codes and the skipped statement, the tests would pass and the correct if/else is being called.